External/FEXCore/Source/Utils/Allocator/IntrusiveArenaAllocator.h

```diff
@@ -7,14 +7,21 @@
 
 #include <bitset>
 #include <cstddef>
+#ifdef TERMUX_BUILD
+#include <experimental/memory_resource>
+#include <experimental/list>
+namespace fex_pmr = std::experimental::pmr;
```

Comment:
How about adding a feature test to see when we can remove this workaround?
```cpp
#if __has_include(<memory_resource>)
#error "Workaround can be removed"
#endif
```

Reply:
Resolved.